src/utils/sanitize-html.js

```diff
@@ -1,4 +1,4 @@
-const DISALLOWED_TAGS = ['script', 'img', 'iframe', 'frame', 'button', 'form'];
+const DISALLOWED_TAGS = ['script', 'img', 'iframe', 'frame', 'button', 'form','pre'];
```

Comment:
Where did this change come from? It is not related to word wrapping..?

Reply:
it is an edge case where <pre></pre> could override and break the CSS

Reply:
removed https://github.com/artificialsolutions/teneo-web-chat/commit/b3ed2933ad3381b42154107e841cdac76a7bb194

Reply:
will remove